workbench/about.html

```diff
@@ -1,129 +1,147 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <link
+      rel="stylesheet"
+      href="/src/renderer/menubar/style.css"
+    />
+    <link
+      rel="stylesheet"
+      href="/src/renderer/menubar/about.css"
+    />
+  </head>
   <body>
-    <h1>About InVEST</h1>
-    <span>
-      version: 
-      <em id='invest-version'></em>
-    </span>
-    <br/><br/>
-
-    Documentation: 
-    <a
-      href="http://releases.naturalcapitalproject.org/invest-userguide/latest/"
-    >
-      http://releases.naturalcapitalproject.org/invest-userguide/latest/
-    </a>
+    <div id="header">
+      <img
+        src="/src/renderer/static/invest-logo.png"
+        width=191
+        height=167
+      />
+      <div id='invest-version'>
+      <p>version:</p>
+      <span id='version-string'></span>
+    </div>
+      <p id='invest-copywrite'>
```

Comment:
This is trivial, but would it be worth using `invest-copyright` as the ID instead?  It'd be same number of letters

Reply:
Thank you, yes spelling words correctly is generally a good thing :)